src/core/Akka.MultiNodeTestRunner.Shared/Discovery.cs

```diff
@@ -17,9 +17,13 @@
 using Xunit.Abstractions;
 using Xunit.Sdk;
 
-namespace Akka.MultiNodeTestRunner
+namespace Akka.MultiNodeTestRunner.Shared
 {
-    internal class Discovery : MarshalByRefObject, IMessageSink, IDisposable
+#if CORECLR
```

Comment:
Actually, that was a mistake, the `NetCore` part was from the other PR to add core support, I merely stripped all the core support and re submitted it.